### ISSUE TITLE
Fairness review on new parser algorithm 

### ISSUE THREAD

**Issue by aoifespenge** [wellcometrust/reach#360]

**Comment by nsorros** [comment]
I think the analysis should be the same as we have done so far, just replication using the new model. We should aim to do it end to end but due to the limited data we might find that we need to annotate more before we are able to in which case we might decide to postpone that for the future and just literally replicate on the existing data.

**Comment by ivyleavedtoadflax** [comment]
Why would we need to label more data @nsorros? Btw I said to @aoifespenge today that I envisage this being another airflow task that is completed at the end of a dag just like the end-to-end evaluation for the more usual metrics. Is that what you had in mind?

**Comment by nsorros** [comment]
> Why would we need to label more data @nsorros? Btw I said to @aoifespenge today that I envisage this being another airflow task that is completed at the end of a dag just like the end-to-end evaluation for the more usual metrics. Is that what you had in mind?

Not a bad idea, we can definitely have it in Airflow as well. Even though the analysis would only change when a new model is deployed.

We would need to label more data because how would you quantify whether the algorithm is biased towards sociology or non english publication if none of your data contains either?

**Comment by ivyleavedtoadflax** [comment]
Sorry I guess my question was more, which data should we label more of? 

I think it would make sense to run the ethical assessment after each update to reach, not just the model, because an improvement to the scraper, or adding a new provider could equally have an impact on the fairness of the whole pipeline.

**Comment by nsorros** [comment]
good point and why not. the data that might need more annotating is the gold data, more titles that are matched to pubmed ids and have the neccesary metadata.

**Comment by ivyleavedtoadflax** [comment]
Agree. More data is good data.

**Comment by ivyleavedtoadflax** [comment]
This is currently blocked by https://github.com/wellcometrust/reach/issues/48
